prow/plugins/config.go

```diff
@@ -437,6 +438,34 @@ type ProjectRepoConfig struct {
 	ProjectColumnMap map[string]string `json:"repo_default_column_map,omitempty"`
 }
 
+// ProjectManager represents the config for the ProjectManager plugin, holding top
+// level config options and a list of Projects
+type ProjectManager struct {
+	OrgRepos map[string]ManagedOrgRepo `json:"org/repos,omitempty"`
```

Comment:
I don't think we want the key to be `"org/repos"` -- maybe have:

```go
type Configuration struct {
    // ...
    ProjectManagementConfig map[string]MangedOrgRepo `json:"project_management,omitempty"`
}
```

?

Reply:
how about "org_repo"?

Reply:
We want to have the key be the _name_ of the thing happening, not what it is. There are many configuration options that are org/repo format.

Reply:
cool, changed it as suggested.